dsl/scripts/jobs.groovy

```diff
@@ -75,13 +75,14 @@ def createCommonPipeline(args){
 def jobs = [
     // Capsule playground
     [
-        name: 'private/cd/capsule-test',
+        name: 'private/cd/vegacapsule-stagnet3',
         useScmDefinition: false,
         parameters: {
             booleanParam('BUILD_CAPSULE', false, h('decide if build vegacapsule from source if false VEGACAPSULE_VERSION will be looked up in releases page', 5))
             stringParam('VEGACAPSULE_VERSION', 'v0.1.0', h('version of vegacapsule'))
```

Comment:
it would be good to provide an option to build capsule from a specific `commit hash`, `tag` or `branch`.